app/src/main/java/com/github/giommok/softwaredevproject/Database.java

```diff
@@ -19,6 +19,21 @@
 }
 
 public class Database {
+
+    public static final String FOLDER = "/";

```

Comment:
I'm not sure about this: instead of a folder string we can just add '/' to the following folder strings (in other words, we can just add '/' at the end of CHILD_USERS, for example)

Reply:
Sometimes the string "users/" was used, and some other only "users" was used. I don't know if we can simply supress the "/" or add it everywhere without trouble

Reply:
It should be fine if we add it everywhere, but anyway you can add it and see if the tests are good.

Reply:
Ok i'll add the "/" to the `CHILD_USERS` string